waiter/test/waiter/scheduler/kubernetes_test.clj

```diff
@@ -1887,14 +1923,6 @@
             instance (pod->ServiceInstance dummy-scheduler pod)]
         (is (= (scheduler/make-ServiceInstance expired-instance-map) instance))))
 
-    (testing "pod with envoy sidecar to instance"
-      (let [dummy-scheduler (assoc base-scheduler :restart-expiry-threshold 10)
-            pod' (merge
-                   (assoc-in pod [:metadata :annotations :waiter/service-port] "8080")
-                   (assoc-in pod [:spec :containers 0 :ports 0 :containerPort] 8081))
-            instance (pod->ServiceInstance dummy-scheduler pod)]
```

Comment:
Note that I also fixed a nasty typo here: `pod` should have been `pod'`; i.e., this test wasn't testing anything. :facepalm: